### ISSUE TITLE
How do I add tabs to TabWidget in DML?

### ISSUE THREAD

**Issue by spikespaz** [buggins/dlangui#569]
I can't figure out how to define tabs as children of TabWidget in DML. I would think it would be something like this, but it's not.

```d
TabWidget {
    TabItem {
        text: "General"
        id: "TAB_GENERAL"

        ChildWidgets ...
    }
    TabItem {
        text: "Settings"
        id: "TAB_SETTINGS"

        ChildWidgets ...
    }
}

**Comment by triplejam** [comment]
I don't think that tabs or docks at the moment can be completely done in DML, not sure. But you can define separate widgets with DML which then can be used in the tabs. Also you could make a widget with DML which contains a VerticalLayout which can hold your TabWidget.
